### ISSUE TITLE
Please consider a WordPress dashboard widget!

### ISSUE THREAD

**Issue by robbie-hunt** [johngodley/redirection#3483]
I'm requesting that at some point could you please consider creating a WordPress dashboard widget for this plugin!

I'm currently using RankMath, and it has a very handy 404 counter that I can see as soon as I log into WordPress as an admin. The issue with using RankMath's 404 counter is that 1. Redirection is the only redirect plugin I use, 2. RankMath's 404 logs aren't comprehensive enough, and 3. any 404 patterns/matches which I have chosen to exclude from the Redirection logs, I also have to exclude from the RankMath logs if I want the counter to exclude them.

It would be excellent if Redirection had a simple counter like the RankMath one, with maybe some of the top URLs to be 404'd and the 404 count of each URL next to it.

This plugin is excellent other than this, thank you very much for creating such a quality plugin.

<img width="614" alt="Screenshot 2022-11-29 at 12 00 42" src="https://user-images.githubusercontent.com/69302142/204524813-2c1ea0f3-fef9-4523-ad4e-3d0363411599.png">

**Comment by johngodley** [comment]
I'm very wary of this because I don't want the 404 log to be a todo list. It's not important that you 'fix' every 404 that occurs on your site. Rank math does a lot of things I don't agree with, and this may be one of them.

**Comment by robbie-hunt** [comment]
You're right. I tend to use the Rank Math counter as a means of seeing trends of bots crawling old links they had previously indexed (the domain has had two or three other websites with terrible links/no linking system), and although I use the Redirection plugin to 'ignore' a lot of 404 links so they don't reappear in the Redirection 404 log, a counter wouldn't be that necessary.
